### ISSUE TITLE
Inconsistent cityscape label encoding

### ISSUE THREAD

**Issue by devinaconley** [divamgupta/image-segmentation-keras#89]
Seems that cityscape labels are encoded differently at various points across this repo
- the pretrained pspnet model outputs labels aligned with the cityscape recommendations
  - https://github.com/mcordts/cityscapesScripts/blob/master/cityscapesscripts/helpers/labels.py
  - e.g. `road`: `0`
- the cityscape preparation script seems to increment all of those values by 1
  - https://github.com/divamgupta/image-segmentation-keras/blob/8ebe98bad4091f8690632c0cf0f25c078ed3788a/scripts/prep_cityscapes_dataset.py#L20
  - e.g. `road`: `1`
- the sample prepared training set follows some other encoding
  - e.g. `road`: `3`

Are the various encoding schemes documented somewhere?
